src/gcp_scanner/crawl.py

```diff
@@ -961,12 +920,8 @@ def get_service_accounts(project_name: str,
     request = service.projects().serviceAccounts().list(name=name)
     while request is not None:
       response = request.execute()
-
-      for service_account in response.get("accounts", []):
-        service_accounts.append(
-            (service_account["email"], service_account["description"]
-             if "description" in service_account else ""))
-
+      service_accounts=list(map(lambda x:(x["email"],x["description"]
```

Comment:
Why did you change this from the list comprehension to this? This is less readable.

Reply:
reverted the map() to list comprehension